openg2p_registration/models/openg2p_beneficiary_school.py

```diff
@@ -0,0 +1,48 @@
+from odoo import api, fields, models
```

Comment:
What is the point of this model if you have already added the fields mentioned here inside the registration model? This just serves as a duplicate.